### ISSUE TITLE
About multi-frame post-training

### ISSUE THREAD

**Issue by zhangziwenHIT** [InternRobotics/CronusVLA#2]
Hello! I have some questions. The paper mentions that during training after multiple frames, the backbone weights for extracting features from historical frames are copied and frozen. The backbone weights for the current frame are updated freely. Doesn't this mean that during inference, two sets of weights need to be loaded to extract features from the historical and current frames respectively? TKS!

<img width="1260" height="423" alt="Image" src="https://github.com/user-attachments/assets/b67d84ac-c165-4406-812e-30d144095d18" />

**Comment by LiHaoHN** [comment]
Thank you for your question!
For multi-frame training, all components of our model (including the V-L backbone parameters) are fully fine-tuned. The only difference is that the gradients of the historical frames are detached, meaning that this part of the data is not used for updating parameters; it serves purely as auxiliary input. During inference, since gradients are not involved and only forward passes are executed, both historical and current information are treated equivalently. Therefore, there’s no need to load two sets of weights.
As for fine-tuning, in our latest version, we still adopt full fine-tuning, following essentially the same process as described above. Please feel free to ask questions🤝🤝, both email and GitHub issues are welcome!

**Comment by zhangziwenHIT** [comment]
> Thank you for your question! For multi-frame training, all components of our model (including the V-L backbone parameters) are fully fine-tuned. The only difference is that the gradients of the historical frames are detached, meaning that this part of the data is not used for updating parameters; it serves purely as auxiliary input. During inference, since gradients are not involved and only forward passes are executed, both historical and current information are treated equivalently. Therefore, there’s no need to load two sets of weights. As for fine-tuning, in our latest version, we still adopt full fine-tuning, following essentially the same process as described above. Please feel free to ask questions🤝🤝, both email and GitHub issues are welcome!

I understand, thank you！
